src/Processors/Formats/Impl/ParallelFormattingOutputFormat.h

```diff
@@ -101,6 +101,9 @@ class ParallelFormattingOutputFormat : public IOutputFormat
         finishAndWait();
     }
 
+    /// There are no formats which support parallel formatting and progress writing at the same time
```

Comment:
Why, by the way? We could probably output the number of rows up to the last processing unit we have put into the queue. It would be slightly further than real progress, but not by much.

Reply:
For now there are only several formats that could display Progress. They are `JSONEachRowWithProgress`, `JSONStringsEachRowWithProgress` and maybe `XML`..

But it is a draft, maybe I will support them.